src/activity_handlers.cpp

```diff
@@ -3955,6 +3955,8 @@ void activity_handlers::spellcasting_finish( player_activity *act, player *p )
                 break;
             case hp_energy:
                 blood_magic( p, cost );
+            case fatigue_energy:
+                p->mod_fatigue( cost );
```

Comment:
Huh. I wonder why Travis and/or Jenkins didn't complain about fallthroughs here and above, but MSYS did?

@jbytheway Do you have any idea?

Reply:
Not sure.  I got the error locally (gcc 7.3).